### PR TITLE
moved to lesson02 root folder

### PR DESCRIPTION
**Pull request by charlycastro**
moving lesson02 files to root folder to comply with others

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,4 +41,7 @@ override.tf.json
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
 
-.env
\ No newline at end of file
+.env
+
+#DS local files
+.DS_Store/*
```
Comment (luisc009) [inline .gitignore:47]: This is related to the local machine, it is better to add it to the global ignore than adding them in all project. 

https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

File: lesson02/main.tf [hunk 2]
```diff
@@ -50,14 +48,20 @@ resource "aws_autoscaling_group" "asg" {
...
+  vpc_zone_identifier       = ["${data.aws_subnet_ids.vpc_subnets.ids}"]
 
-  min_size              = "${var.min_size}"
-  max_size              = "${var.max_size}"
-  wait_for_elb_capacity = "${var.min_size}"
-
-  desired_capacity = "${var.desired_capacity}"
+  //TAGS propagated to each EC2 instance
+  tags = "${list(
+    map("key", "Name",          "value", "${var.metadata["appname"]}-${var.env}-ec2-${var.metadata["appversion"]}",         "propagate_at_launch", true)
```
Comment (chitopunk) [inline lesson02/main.tf]: is this ok? `map("key", "Name",          "value",` with a lot of spaces?

Comment (charlycastro) [reply lesson02/main.tf]: let me move them

File: .gitignore (hunk 1 above)
Comment (charlycastro) [reply .gitignore:47]: i think both, if the contributor doesn't have this in their local we will still get them :D

File: lesson02/main.tf (hunk 2 above)
Comment (charlycastro) [reply lesson02/main.tf]: @chitopunk

**Review by chitopunk — APPROVED** [review]

**Pull request merged**